lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -85,24 +85,37 @@
  * @TODO: move into Forge pattern library if approved.
  * Add to the _regions/_container pattern.
  */
-.container__block  {
-  &.-primary {
-    @include media($tablet) {
-      @include span(6 of 12)
-    }
-
-    @include media($desktop) {
-      @include span(8 of 12)
-    }
-  }
+.container {
+  > .wrapper {
 
-  &.-secondary {
-    @include media($tablet) {
-      @include span(6 of 12)
+    .container__block  {
+      &.-primary {
+        @include media($tablet) {
+          @include span(6 of 12)
+        }
+
+        @include media($desktop) {
+          @include span(8 of 12)
+        }
+      }
+
+      &.-secondary {
+        @include media($tablet) {
+          @include span(6 of 12)
+        }
+
+        @include media($desktop) {
+          @include span(4 of 12)
+        }
+      }
+
+      &.-narrow {
+        &.-centered {
```

Comment:
Yeah, I wasn't a fan of this, but needed it based on the designs that now included a narrow block that is centered. I couldn't get it working without the specificity 😞